src/dune_lang/syntax.ml

```diff
@@ -71,8 +71,10 @@ type t =
 
 module Error_msg = struct
   let since t ver ~what =
-    Printf.sprintf "%s is only available since version %s of %s" what
-      (Version.to_string ver) t.desc
+    Printf.sprintf
+      "%s is only available since version %s of %s. Please update your \
+       `dune-project` file to have `(lang %s)`."
```

Comment:
Just a nitpick about the use of backticks: we don't use markdown syntax in messages produced by Dune, so to be consistent we shouldn't use it here either. I suggest to just remove the backticks altogether. Alternatively, we can replace them by simple or double quotes, but here that should be clear enough.

Reply:
Ah, cool. Thanks for the review!

Reply:
No problem, thanks for the PR! :)